test/prog/dftb+/tests

```diff
@@ -202,6 +202,8 @@ solvation/gbsa_param3                  #? MPI_PROCS <= 2
 solvation/alpb_cation                  #? MPI_PROCS <= 1
 solvation/alpb_anion                   #? MPI_PROCS <= 1
 solvation/alpb_p16                     #? MPI_PROCS <= 1
+solvation/cosmo_g1                     #? MPI_PROCS <= 2
+solvation/comso_b33                    #? MPI_PROCS <= 2
```

Comment:
Should be `cosmo` instead of `comso`. (Unfortunately, our tester silently ignores this. Would have to be fixed separately...)

Reply:
My bad, didn't noticed that typo. I was always hoping the testsuite would bark at me when I include wrong names, hehe.